src/components/course/CourseRunSelector.jsx

```diff
@@ -32,22 +32,24 @@ export default function CourseRunSelector() {
       );
     }
     return (
-      <div className="input-group">
+      <div className="d-flex align-middle">
         <Input
           name="courseRun"
-          className="col-4"
+          className="col-4 mb-2"
```

Comment:
should we also add `mb-2` to the "more dates" button so this spacing is there as well?

Reply:
More dates actually looks pretty good without it but let me know whatcha think
<img width="369" alt="Screen Shot 2020-06-19 at 9 15 34 AM" src="https://user-images.githubusercontent.com/1559107/85136351-7d9d4800-b20d-11ea-9d52-dd0ad733b63e.png">

Reply:
Hmm, playing around with devtools on Chrome (on stage), I think adding `mb-1` would be worthwhile to give just a hair more breathing room. But agree `mb-2` is a bit much.